src/js/index.js

```diff
@@ -1 +1,64 @@
 import '../sass/style.scss';
+
+// todo:-----------> Get data from API
+
+const baseUrl = 'https://api.tvmaze.com/shows/';
+
+async function makeShowUrl(id) {
+  const temp = fetch(`${baseUrl}${id}`);
+  const temp1 = await temp;
+  const temp2 = temp1.json();
+  return temp2;
+}
+
+// todo:-----------> Show them on HTML (name + image)
+
+function makeElementsForShow() {
+  const createLi = document.createElement('li');
+  const createImg = document.createElement('img');
+  const createDiv = document.createElement('div');
```

Comment:
- [ ]  it may be better if we use template literal so that our code will be shorter and maintainable.